cfme/tests/automate/test_service_automate.py

```diff
@@ -378,8 +381,8 @@ def test_service_retire_automate():
 
 
 @pytest.mark.tier(2)
-@pytest.mark.meta(coverage=[1740796])
-def test_import_dialog_file_without_selecting_file():
+@pytest.mark.meta(coverage=[1740796], blockers=[BZ(1740796, forced_streams=["5.10"])])
```

Comment:
This test case is not automating the BZ yet , hence no need to block . 
If you are automating the BZ then you have to check that the flash message is not shown after selecting a dialog and editing it  and then the blocker is needed. 

Also , you need to add `automated` instead of `coverage`

Reply:
This test case is automating this BZ by checking service dialog update.
I am changing coverage to automates